mne/io/edf/edf.py

```diff
@@ -75,6 +75,9 @@ class RawEDF(BaseRaw):
         For unknown prefixes, the type will be 'EEG' and the name will not be
         modified. If False, do not infer types and assume all channels are of
         type 'EEG'.
+    picks : list of str | str
+        Channel names to be picked. Channels in picks are picked even if they
+        are in exclude.
```

Comment:
I am not sure it's the route I would have followed. Basically you can already do what you want by putting in exclude all but the "picks" here. Also would 

`read_raw_edf(fname, preload=False, picks==picks)`

be equivalent to:

`read_raw_edf(fname, preload=False).pick(picks)`

?

Reply:
I agree that `exclude` seems like it already covers the stated use case (preventing from loading one or a few channels with high sampling frequencies). @rcmdnk is there a reason that won't work for you?

Reply:
For example, if the EDF has channels A and B, which have 100 Hz and 200 Hz frequency, respectively.

In below cases, `raw.info['sfreq']` are different:

```python
raw = read_raw_edf(fname, preload=False, picks="A")
print(raw.info['sfreq'])
```
shows `100`.

```python
raw = read_raw_edf(fname, preload=False).picks("A")
print(raw.info['sfreq'])
```
shows `200`.

In the second case, `raw.resample(sfreq=100)` works to make it similar to the first data, but it is also a bit different from it because of resampling options.

Reply:
@drammock 
Yes, `exclude` can work if you can write all channels to be excluded.

But most of the time, in my usage, I would like to choose one or a few channels from the EDF in which 10 or 20 channels exist.
In this case, it's much easier to write `picks` rather than listing out all excluded channels.

In addition, sometimes EDF files have additional channels even when they are in the same group (I found some such files in the open EDF data).
The main channel names for EEG or EOG are always fixed.

So, sometimes it is easier and safer to use `picks`.